src/main/java/Task3.java

```diff
@@ -0,0 +1,71 @@
+import java.util.Scanner;
+import java.text.DecimalFormat;
+import java.lang.Math;
+
+public class Task3 {
+    public static void main(String[] args) {
+        String s1 = getInput("Enter a numeric value: ");
+        String s2 = getInput("Enter a numeric value: ");
+        String op = getInput("Choose an operation (+ - * / %): ");
+
+        int result = 0;
+
+        switch (op) {
+            case "+":
+                result = (int) addValues(s1, s2);
+                break;
+            case "-":
+                result = (int) subtractValues(s1, s2);
+                break;
+            case "*":
+                result = (int) multiplyValues(s1, s2);
+                break;
+            case "/":
+                result = (int) divideValues(s1, s2);
+                break;
+            case "%":
+                result = (int) remindValues(s1, s2);
+                break;
+            default:
+                System.out.println("Unrecognized operation!");
+                return;
+        }
+
+        System.out.println("The answer (integer) is " + result);
+    }
+
+    private static String getInput(String prompt) {
+        System.out.print(prompt);
+        Scanner sc = new Scanner(System.in);
+        return sc.nextLine();
+    }
+
+    public static double addValues(String s1, String s2) {
```

Comment:
It would be better to have arguments as Integer (or Double) and not to parse them inside the method. for your main() method you can convert s1 to some number there and then use for switch (or use scanner and then Scanner.nextInt()/Scanner.nextDouble() )

Reply:
Done. input strings converted to double inside main(). No extra conversion inside mathematical methods